tests/functional/spec/checkbox.spec.js

```diff
@@ -39,6 +39,13 @@ describe('Checkbox with "other" option', () => {
     expect($(MandatoryCheckboxPage.error()).isDisplayed()).to.be.true;
   });
 
+  it("Given a mandatory checkbox answer, When I leave the input field empty and submit, Then an error containing the question text should be displayed.", () => {
+    // When
+    $(MandatoryCheckboxPage.submit()).click();
+    // Then
+    expect($(MandatoryCheckboxPage.error()).getText()).to.contain("Select at least one answer");
+  });
```

Comment:
The expect doesn't match the description as it's not checking for the question text in the error message.